cluster_test.go

```diff
@@ -1693,20 +1693,47 @@ func (s *S) TestMaxSocketUses(c *C) {
 		time.Sleep(5e8)
 	}
 
-	// a socket was used during connection
-	numUses := 1
+	for i := 0; i < 2; i++ {
+		stats = mgo.GetStats()
+		// every 4th usage should result in a new connection
```

Comment:
Comment is no longer valid I think. My interpretation is that the test is converted to only test for secs? Which leads credence to the notion that the other option is intended to be removed entirely.

@wgallagher  Are we comfortable removing the existing options for maxSocketUses? Any fear of breaking existing users?